openl3/models.py

```diff
@@ -49,17 +93,25 @@ def load_audio_embedding_model(input_repr, content_type, embedding_size):
         Type of content used to train embedding.
     embedding_size : 6144 or 512
         Embedding dimensionality.
+    frontend : "kapre" or "librosa" (or True or False, respectively)
+        The audio frontend to use.
 
     Returns
     -------
-    model : keras.models.Model
+    model : tf.keras.Model
         Model object.
     """
+    # Semantically, I think booleans make more sense here than kapre/librosa, but for compatibility
+    if frontend is True:
+        frontend = 'kapre'
+    if frontend is False or frontend is None:
+        frontend = 'librosa'
+    frontend = _validate_audio_frontend(frontend)
```

Comment:
I get what you mean and I'm not opposed to removing this. 

The reason that I said that it would make more sense is that it's a yes or no question of whether you want to have the frontend when you load the model. Originally I had the parameter as `include_frontend=True`, but I changed it to be more consistent.

I can remove the parameter aliases tho.